cmd/utils.go

```diff
@@ -1045,3 +1045,22 @@ func checkTime() {
 		}
 	}
 }
+
+func setNewIndexURL() {
+
+	var repoFile = getRepoFileLocation()
+	var oldIndexURL = "https://raw.githubusercontent.com/appsody/stacks/master/index.yaml"
+	var newIndexURL = "https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml"
+
+	data, err := ioutil.ReadFile(repoFile)
+	if err != nil {
+		Warning.log("Unable to read repository file")
+	}
+
+	replaceURL := bytes.Replace(data, []byte(oldIndexURL), []byte(newIndexURL), -1)
+
+	if err = ioutil.WriteFile(repoFile, replaceURL, 0666); err != nil {
```

Comment:
Also, I suggest using 0644 to be consistent with the way we write the repo file today.
See https://github.com/appsody/appsody/blob/master/cmd/repo.go#L476

Reply:
change addressed